lifecycle-operator/controllers/lifecycle/keptnworkloadversion/reconcile_prepostevaluation.go

```diff
@@ -1,16 +1,16 @@
 //nolint:dupl
-package keptnworkloadinstance
+package keptnworkloadversion
 
 import (
 	"context"
 	"fmt"
 
-	klcv1alpha3 "github.com/keptn/lifecycle-toolkit/lifecycle-operator/apis/lifecycle/v1alpha3"
 	apicommon "github.com/keptn/lifecycle-toolkit/lifecycle-operator/apis/lifecycle/v1alpha3/common"
+	klcv1alpha4 "github.com/keptn/lifecycle-toolkit/lifecycle-operator/apis/lifecycle/v1alpha4"
 	controllercommon "github.com/keptn/lifecycle-toolkit/lifecycle-operator/controllers/common"
 )
 
-func (r *KeptnWorkloadInstanceReconciler) reconcilePrePostEvaluation(ctx context.Context, phaseCtx context.Context, workloadInstance *klcv1alpha3.KeptnWorkloadInstance, checkType apicommon.CheckType) (apicommon.KeptnState, error) {
+func (r *KeptnWorkloadVersionReconciler) reconcilePrePostEvaluation(ctx context.Context, phaseCtx context.Context, workloadVersion *klcv1alpha4.KeptnWorkloadVersion, checkType apicommon.CheckType) (apicommon.KeptnState, error) {
 	evaluationHandler := controllercommon.EvaluationHandler{
```

Comment:
this we might want to refactor (of course in another ticket) - would be better for the reconciler to have an injectable EvaluationHandler interface. I will make a ticket for that

Reply:
sure, we have a refactoring ticket here already https://github.com/keptn/lifecycle-toolkit/issues/2261

Reply:
@bacherfl what about adding this to the controller refactoring ticket? https://github.com/keptn/lifecycle-toolkit/issues/2261

Reply:
but could also bee a separate good first issue 👍

Reply:
https://github.com/keptn/lifecycle-toolkit/issues/2268